util/bloomchain/src/group/position/manager.rs

```diff
@@ -0,0 +1,28 @@
+use super::{Position, GroupPosition};
+use position::Position as BloomPosition;
+
+pub struct Manager {
+	index_size: usize
+}
+
+impl Manager {
+	pub fn new(index_size: usize) -> Self {
+		Manager {
+			index_size: index_size
+		}
+	}
+
+	pub fn group_position(&self, pos: &BloomPosition) -> GroupPosition {
+		GroupPosition {
+			level: pos.level,
+			index: pos.index / self.index_size,
```

Comment:
Still trying to get the whole picture - is there some check if `index_size` is not a zero? Does it worth adding a check/assert to this library? Or should it be managed outside?

Reply:
so it's exactly the same library and logic we use today and we've been using for last 2 years. `index_size` is a constant set in ethcore and is always 16. But you are right, there should be an assertion here :/

Reply:
Don't bother with this then - I'm just checking for possible issues. If that is not an issue - just ignore, pelase